vmcnet/models/construct.py

```diff
@@ -638,6 +638,7 @@ def get_residual_blocks_for_ferminet_backflow(
                 use_bias,
                 skip_connection=two_electron_skip,
                 skip_connection_scale=two_electron_skip_scale,
+                use_transformer=use_transformer,
```

Comment:
Hm don't we need a separate flag for controlling the transformer setting in the 1e stream vs 2e stream?

It seems like currently if we set use_transformer=True, the 1e stream will stop listening to the 2e stream, and it will be irrelevant that the 2e stream is using a transformer.